### PR TITLE
Update Hungarian lang files

### PR DESCRIPTION
**Pull request by yesszus**

### CODE REVIEW

**Comment by yesszus** [comment]
@lonalore pls review this.

File: Hungarian/e107_plugins/news/news_months_menu.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<?php
```
Comment (lonalore) [inline Hungarian/e107_plugins/news/news_months_menu.php]: @yesszus what about this file? I think this is the wrong place... Please remove it. The rest is OK.

Comment (yesszus) [reply Hungarian/e107_plugins/news/news_months_menu.php]: Deleted!

**Comment by lonalore** [comment]
Thank you!!

**Pull request merged**